packages/server-wallet/src/wallet/__test__/integration/join-channel.test.ts

```diff
@@ -0,0 +1,68 @@
+import {simpleEthAllocation} from '@statechannels/wallet-core';
+
+import {Channel} from '../../../models/channel';
+import {Wallet} from '../..';
+import {seed} from '../../../db/seeds/1_signing_wallet_seeds';
```

Comment:
Maybe do `seed as seedSigningWallets` to be more more explicit?

Reply:
https://github.com/statechannels/statechannels/pull/2375/commits/dff9037f149ec351d758f508dc939b0bfa1794ec